src/main/features/eligibility/store.ts

```diff
@@ -0,0 +1,31 @@
+import { Eligibility } from 'eligibility/model/eligibility'
+import * as Cookies from 'cookies'
+import * as express from 'express'
+
+const minuteInMilliseconds = 60 * 1000
+const cookieTimeToLiveInMinutes = 10 * minuteInMilliseconds
+
+export const cookieName = 'eligibility-check'
+
+export interface EligibilityStore {
```

Comment:
This interface is implemented by one class only, I think it can be removed.

Reply:
Removed